[ffmpeg] Static build error (libopenmpt >= 0.2.6557 not found using pkg-config)

Package: ffmpeg[all,all-gpl,all-nonfree,amf,aom,ass,avcodec,avdevice,avfilter,avformat,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openmpt,openssl,opus,postproc,qsv,sdl2,snappy,soxr,speex,srt,ssh,swresample,swscale,theora,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-windows-static -> 5.1.2#4

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.35.32215.0
-    vcpkg-tool version: 2023-02-16-12e657924d99511514c0287ca5ce46882d3657c7
    vcpkg-scripts version: 8daf70c56 2023-03-03 (2 days ago)

**To Reproduce**

`vcpkg install ffmpeg[all-nonfree,ffmpeg,ffprobe,ffplay]:x64-windows-static`

**Workaround**

A "workaround" for building `ffmpeg` is to remove `openmpt` from ffmpeg dependencies.

**Failure**

This sounds to be the main/base issue (more info in the log above) while building ffmpeg:
```
ERROR: libopenmpt >= 0.2.6557 not found using pkg-config
```

**Failure logs**

```
-- Downloading https://github.com/ffmpeg/ffmpeg/archive/n5.1.2.tar.gz -> ffmpeg-ffmpeg-n5.1.2.tar.gz...
-- Extracting source E:/vc/downloads/ffmpeg-ffmpeg-n5.1.2.tar.gz
-- Applying patch 0001-create-lib-libraries.patch
-- Applying patch 0002-fix-msvc-link.patch
-- Applying patch 0003-fix-windowsinclude.patch
-- Applying patch 0004-fix-debug-build.patch
-- Applying patch 0005-fix-nasm.patch
-- Applying patch 0006-fix-StaticFeatures.patch
-- Applying patch 0007-fix-lib-naming.patch
-- Applying patch 0009-Fix-fdk-detection.patch
-- Applying patch 0011-Fix-x265-detection.patch
-- Applying patch 0012-Fix-ssl-110-detection.patch
-- Applying patch 0013-define-WINVER.patch
-- Applying patch 0015-Fix-xml2-detection.patch
-- Applying patch 0020-fix-aarch64-libswscale.patch
-- Applying patch 0022-fix-iconv.patch
-- Using source at E:/d/ffmpeg/src/n5.1.2-f31542651f.clean
-- Downloading https://www.nasm.us/pub/nasm/releasebuilds/2.16.01/win64/nasm-2.16.01-win64.zip -> nasm-2.16.01-win64.zip...
-- Downloading https://repo.msys2.org/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/gzip-1.11-1-x86_64.pkg.tar.zst -> msys-gzip-1.11-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/bash-5.1.008-1-x86_64.pkg.tar.zst -> msys-bash-5.1.008-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/diffutils-3.8-2-x86_64.pkg.tar.zst -> msys-diffutils-3.8-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/coreutils-8.32-2-x86_64.pkg.tar.zst -> msys-coreutils-8.32-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://mirror.bit.edu.cn/msys2/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://mirror.selfnet.de/msys2/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz -> msys-grep-3.0-2-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/sed-4.8-2-x86_64.pkg.tar.zst -> msys-sed-4.8-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libpcre-8.45-1-x86_64.pkg.tar.zst -> msys-libpcre-8.45-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/gawk-5.1.0-2-x86_64.pkg.tar.zst -> msys-gawk-5.1.0-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst -> msys-mpfr-4.1.0-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/gmp-6.2.1-1-x86_64.pkg.tar.zst -> msys-gmp-6.2.1-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libreadline-8.1.001-1-x86_64.pkg.tar.zst -> msys-libreadline-8.1.001-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/ncurses-6.2-2-x86_64.pkg.tar.zst -> msys-ncurses-6.2-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/automake1.16-1.16.3-3-any.pkg.tar.zst -> msys-automake1.16-1.16.3-3-any.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/perl-5.32.1-2-x86_64.pkg.tar.zst -> msys-perl-5.32.1-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libcrypt-2.1-3-x86_64.pkg.tar.zst -> msys-libcrypt-2.1-3-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/pkg-config-0.29.2-4-x86_64.pkg.tar.zst -> msys-pkg-config-0.29.2-4-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/make-4.3-3-x86_64.pkg.tar.zst -> msys-make-4.3-3-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libintl-0.21-1-x86_64.pkg.tar.zst -> msys-libintl-0.21-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst -> msys-libiconv-1.16-2-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/gcc-libs-11.2.0-3-x86_64.pkg.tar.zst -> msys-gcc-libs-11.2.0-3-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://mirror.selfnet.de/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst -> msys-msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst...
-- Applying patch E:/vc/scripts/msys/compile_wrapper_consider_clang-cl.patch
-- Using msys root at E:/vc/downloads/tools/msys2/893f8801e7c30cbd
-- Found external ninja('1.11.0').
-- Getting CMake variables for x64-windows-static
-- Building Options: --toolchain=msvc --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --disable-autodetect --pkg-config=E:/vc/installed/x64-windows/tools/pkgconf/pkgconf.exe --target-os=win32 --enable-w32threads --enable-d3d11va --enable-dxva2 --enable-mediafoundation --disable-inline-asm --cc=cl.exe --cxx=cl.exe --windres=rc.exe --ld=link.exe --ar='ar-lib lib.exe' --enable-nonfree --enable-gpl --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avformat --enable-avfilter --enable-postproc --enable-swresample --enable-swscale --disable-alsa --enable-amf --enable-libaom --enable-libass --disable-avisynth --enable-bzlib --enable-libdav1d --enable-libfdk-aac --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-iconv --enable-libilbc --enable-lzma --enable-libmp3lame --enable-libmodplug --enable-cuda --enable-nvenc --enable-nvdec --enable-cuvid --enable-ffnvcodec --enable-opencl --enable-opengl --enable-libopenh264 --enable-libopenjpeg --enable-libopenmpt --enable-openssl --enable-libopus --enable-sdl2 --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --disable-libtensorflow --disable-libtesseract --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-zlib --enable-libsrt --enable-libmfx --enable-encoder=h264_qsv --enable-decoder=h264_qsv  --enable-cross-compile --extra-cflags=-DHAVE_UNISTD_H=0 --pkg-config-flags=--static
-- Building Release Options: --enable-optimizations
-- Building ffmpeg for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: E:/vc/downloads/tools/msys2/893f8801e7c30cbd/usr/bin/bash.exe ./build.sh
    Working Directory: E:/d/ffmpeg/x64-windows-static-rel
    Error code: 1
    See logs for more information:
      E:\d\ffmpeg\build-x64-windows-static-rel-config.log
      E:\d\ffmpeg\build-x64-windows-static-rel-out.log

Call Stack (most recent call first):
  ports/ffmpeg/portfile.cmake:596 (vcpkg_execute_required_process)
  scripts/ports.cmake:147 (include)



```
<details><summary>E:\d\ffmpeg\build-x64-windows-static-rel-config.log</summary>

```
# /e/d/ffmpeg/src/n5.1.2-f31542651f.clean/configure --prefix=/e/vc/packages/ffmpeg_x64-windows-static --toolchain=msvc --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --disable-autodetect --pkg-config='E:/vc/installed/x64-windows/tools/pkgconf/pkgconf.exe' --target-os=win32 --enable-w32threads --enable-d3d11va --enable-dxva2 --enable-mediafoundation --disable-inline-asm --cc=cl.exe --cxx=cl.exe --windres=rc.exe --ld=link.exe --ar='ar-lib lib.exe' --enable-nonfree --enable-gpl --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avformat --enable-avfilter --enable-postproc --enable-swresample --enable-swscale --disable-alsa --enable-amf --enable-libaom --enable-libass --disable-avisynth --enable-bzlib --enable-libdav1d --enable-libfdk-aac --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-iconv --enable-libilbc --enable-lzma --enable-libmp3lame --enable-libmodplug --enable-cuda --enable-nvenc --enable-nvdec --enable-cuvid --enable-ffnvcodec --enable-opencl --enable-opengl --enable-libopenh264 --enable-libopenjpeg --enable-libopenmpt --enable-openssl --enable-libopus --enable-sdl2 --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --disable-libtensorflow --disable-libtesseract --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-zlib --enable-libsrt --enable-libmfx --enable-encoder=h264_qsv --enable-decoder=h264_qsv --enable-cross-compile --extra-cflags='-DHAVE_UNISTD_H=0' --pkg-config-flags=--static --enable-optimizations --arch=x86_64 --enable-asm --enable-x86asm
ALLUSERSPROFILE='C:\ProgramData'
ALL_COMPONENTS='
    
    aac_adtstoasc_bsf
av1_frame_merge_bsf
av1_frame_split_bsf
av1_metadata_bsf
chomp_bsf
dump_extradata_bsf
dca_core_bsf
dv_error_marker_bsf
eac3_core_bsf
extract_extradata_bsf
filter_units_bsf
h264_metadata_bsf
h264_mp4toannexb_bsf
h264_redundant_pps_bsf
hapqa_extract_bsf
hevc_metadata_bsf
hevc_mp4toannexb_bsf
imx_dump_header_bsf
mjpeg2jpeg_bsf
mjpega_dump_header_bsf
mp3_header_decompress_bsf
mpeg2_metadata_bsf
mpeg4_unpack_bframes_bsf
mov2textsub_bsf
noise_bsf
null_bsf
opus_metadata_bsf
pcm_rechunk_bsf
pgs_frame_merge_bsf
prores_metadata_bsf
remove_extradata_bsf
setts_bsf
text2movsub_bsf
trace_headers_bsf
truehd_core_bsf
vp9_metadata_bsf
vp9_raw_reorder_bsf
vp9_superframe_bsf
vp9_superframe_split_bsf
    aasc_decoder
aic_decoder
alias_pix_decoder
agm_decoder
amv_decoder
anm_decoder
ansi_decoder
apng_decoder
arbc_decoder
argo_decoder
asv1_decoder
asv2_decoder
aura_decoder
aura2_decoder
avrp_decoder
avrn_decoder
avs_decoder
avui_decoder
ayuv_decoder
bethsoftvid_decoder
bfi_decoder
bink_decoder
bitpacked_decoder
bmp_decoder
bmv_video_decoder
brender_pix_decoder
c93_decoder
cavs_decoder
cdgraphics_decoder
cdtoons_decoder
cdxl_decoder
cfhd_decoder
cinepak_decoder
clearvideo_decoder
cljr_decoder
cllc_decoder
comfortnoise_decoder
cpia_decoder
cri_decoder
cscd_decoder
...
Skipped 18145 lines
...

libmp3lame-static.lib(mpglib_interface.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(mpglib_interface.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(encoder.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(encoder.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(id3tag.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(id3tag.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(gain_analysis.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(gain_analysis.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(quantize_pvt.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(quantize_pvt.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(psymodel.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(psymodel.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(VbrTag.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(VbrTag.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(tables.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(tables.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(takehiro.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(takehiro.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(newmdct.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(newmdct.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(quantize.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(quantize.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(reservoir.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(reservoir.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(fft.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(fft.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(vbrquantize.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(vbrquantize.obj)' or at ''; linking object as if no debug info
libmp3lame-static.lib(xmm_quantize_sub.obj) : warning LNK4099: PDB '' was not found with 'libmp3lame-static.lib(xmm_quantize_sub.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(interface.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(interface.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(common.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(common.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(tabinit.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(tabinit.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(layer3.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(layer3.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(decode_i386.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(decode_i386.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(layer1.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(layer1.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(layer2.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(layer2.obj)' or at ''; linking object as if no debug info
libmpghip-static.lib(dct64_i386.obj) : warning LNK4099: PDB '' was not found with 'libmpghip-static.lib(dct64_i386.obj)' or at ''; linking object as if no debug info
require_pkg_config libopenh264 openh264 wels/codec_api.h WelsGetCodecVersion
check_pkg_config libopenh264 openh264 wels/codec_api.h WelsGetCodecVersion
test_pkg_config libopenh264 openh264 wels/codec_api.h WelsGetCodecVersion
E:/vc/installed/x64-windows/tools/pkgconf/pkgconf.exe --exists --print-errors openh264
check_func_headers wels/codec_api.h WelsGetCodecVersion -IE:/vc/installed/x64-windows-static/include -LE:/vc/installed/x64-windows-static/lib -lopenh264 -lstdc++
test_ld cc -IE:/vc/installed/x64-windows-static/include -LE:/vc/installed/x64-windows-static/lib -lopenh264 -lstdc++
test_cc -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib
BEGIN ./ffconf.b4qVxAkc/test.c
    1	#include <wels/codec_api.h>
    2	#include <stdint.h>
    3	long check_WelsGetCodecVersion(void) { return (long) WelsGetCodecVersion; }
    4	int main(void) { int ret = 0;
    5	 ret |= ((intptr_t)check_WelsGetCodecVersion) & 0xFFFF;
    6	return ret; }
END ./ffconf.b4qVxAkc/test.c
cl.exe -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_USE_MATH_DEFINES -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS -D_WIN32_WINNT=0x0600 -DPIC @E:/d/ffmpeg/x64-windows-static-rel/cflags.rsp -nologo -DHAVE_UNISTD_H=0 -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/harfbuzz -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib -c -Fo./ffconf.b4qVxAkc/test.o ./ffconf.b4qVxAkc/test.c
cl : Command line warning D9002 : ignoring unknown option '-libpath:E:/vc/installed/x64-windows-static/lib'
test.c
./ffconf.b4qVxAkc/test.c(3): warning C4311: 'type cast': pointer truncation from 'OpenH264Version (__cdecl *)(void)' to 'long'
link.exe @E:/d/ffmpeg/x64-windows-static-rel/ldflags.rsp -nologo -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib -out:./ffconf.b4qVxAkc/test.exe ./ffconf.b4qVxAkc/test.o openh264.lib
LINK : warning LNK4044: unrecognized option '/IE:/vc/installed/x64-windows-static/include'; ignored
check_pkg_config libopenjpeg libopenjp2 >= 2.1.0 openjpeg.h opj_version
test_pkg_config libopenjpeg libopenjp2 >= 2.1.0 openjpeg.h opj_version
E:/vc/installed/x64-windows/tools/pkgconf/pkgconf.exe --exists --print-errors libopenjp2 >= 2.1.0
check_func_headers openjpeg.h opj_version -IE:/vc/installed/x64-windows-static/include -LE:/vc/installed/x64-windows-static/lib -lopenjp2
test_ld cc -IE:/vc/installed/x64-windows-static/include -LE:/vc/installed/x64-windows-static/lib -lopenjp2
test_cc -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib
BEGIN ./ffconf.b4qVxAkc/test.c
    1	#include <openjpeg.h>
    2	#include <stdint.h>
    3	long check_opj_version(void) { return (long) opj_version; }
    4	int main(void) { int ret = 0;
    5	 ret |= ((intptr_t)check_opj_version) & 0xFFFF;
    6	return ret; }
END ./ffconf.b4qVxAkc/test.c
cl.exe -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_USE_MATH_DEFINES -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS -D_WIN32_WINNT=0x0600 -DPIC @E:/d/ffmpeg/x64-windows-static-rel/cflags.rsp -nologo -DHAVE_UNISTD_H=0 -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/harfbuzz -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib -c -Fo./ffconf.b4qVxAkc/test.o ./ffconf.b4qVxAkc/test.c
cl : Command line warning D9002 : ignoring unknown option '-libpath:E:/vc/installed/x64-windows-static/lib'
test.c
./ffconf.b4qVxAkc/test.c(3): warning C4311: 'type cast': pointer truncation from 'const char *(__cdecl *)(void)' to 'long'
link.exe @E:/d/ffmpeg/x64-windows-static-rel/ldflags.rsp -nologo -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib -out:./ffconf.b4qVxAkc/test.exe ./ffconf.b4qVxAkc/test.o openjp2.lib
LINK : warning LNK4044: unrecognized option '/IE:/vc/installed/x64-windows-static/include'; ignored
require_pkg_config libopenmpt libopenmpt >= 0.2.6557 libopenmpt/libopenmpt.h openmpt_module_create -lstdc++
check_pkg_config libopenmpt libopenmpt >= 0.2.6557 libopenmpt/libopenmpt.h openmpt_module_create -lstdc++
test_pkg_config libopenmpt libopenmpt >= 0.2.6557 libopenmpt/libopenmpt.h openmpt_module_create -lstdc++
E:/vc/installed/x64-windows/tools/pkgconf/pkgconf.exe --exists --print-errors libopenmpt >= 0.2.6557
check_func_headers libopenmpt/libopenmpt.h openmpt_module_create -IE:/vc/installed/x64-windows-static/include -LE:/vc/installed/x64-windows-static/lib -lopenmpt -lzlib -lvorbisfile -lvorbis -logg -lmpg123 -lstdc++
test_ld cc -IE:/vc/installed/x64-windows-static/include -LE:/vc/installed/x64-windows-static/lib -lopenmpt -lzlib -lvorbisfile -lvorbis -logg -lmpg123 -lstdc++
test_cc -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib
BEGIN ./ffconf.b4qVxAkc/test.c
    1	#include <libopenmpt/libopenmpt.h>
    2	#include <stdint.h>
    3	long check_openmpt_module_create(void) { return (long) openmpt_module_create; }
    4	int main(void) { int ret = 0;
    5	 ret |= ((intptr_t)check_openmpt_module_create) & 0xFFFF;
    6	return ret; }
END ./ffconf.b4qVxAkc/test.c
cl.exe -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_USE_MATH_DEFINES -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS -D_WIN32_WINNT=0x0600 -DPIC @E:/d/ffmpeg/x64-windows-static-rel/cflags.rsp -nologo -DHAVE_UNISTD_H=0 -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/harfbuzz -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include/libpng16 -IE:/vc/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib -c -Fo./ffconf.b4qVxAkc/test.o ./ffconf.b4qVxAkc/test.c
cl : Command line warning D9002 : ignoring unknown option '-libpath:E:/vc/installed/x64-windows-static/lib'
test.c
./ffconf.b4qVxAkc/test.c(3): warning C4311: 'type cast': pointer truncation from 'openmpt_module *(__cdecl *)(openmpt_stream_callbacks,void *,openmpt_log_func,void *,const openmpt_module_initial_ctl *)' to 'long'
./ffconf.b4qVxAkc/test.c(3): warning C4996: 'openmpt_module_create': was declared deprecated
link.exe @E:/d/ffmpeg/x64-windows-static-rel/ldflags.rsp -nologo -IE:/vc/installed/x64-windows-static/include -libpath:E:/vc/installed/x64-windows-static/lib -out:./ffconf.b4qVxAkc/test.exe ./ffconf.b4qVxAkc/test.o openmpt.lib zlib.lib vorbisfile.lib vorbis.lib ogg.lib mpg123.lib
LINK : warning LNK4044: unrecognized option '/IE:/vc/installed/x64-windows-static/include'; ignored
mpg123.lib(compat.c.obj) : error LNK2019: unresolved external symbol __imp_PathCombineW referenced in function INT123_compat_catpath
mpg123.lib(compat.c.obj) : error LNK2019: unresolved external symbol __imp_PathIsRelativeW referenced in function u2wlongpath
mpg123.lib(compat.c.obj) : error LNK2019: unresolved external symbol __imp_PathIsUNCW referenced in function u2wlongpath
.\ffconf.b4qVxAkc\test.exe : fatal error LNK1120: 3 unresolved externals
ERROR: libopenmpt >= 0.2.6557 not found using pkg-config
```
</details>
<details><summary>E:\d\ffmpeg\build-x64-windows-static-rel-out.log</summary>

```
BUILD_ARCH=x86_64
OPTION_VARIABLE=OPTIONS_x86_64
=== CONFIGURING ===
ERROR: libopenmpt >= 0.2.6557 not found using pkg-config

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.libera.chat.
Include the log file "ffbuild/config.log" produced by configure as this will help
solve the problem.
```
</details>

**Additional context**
- Build has been started on a fresh Windows 10 Pro 22H2 installation
- Visual Studio Community 2022
- Windows 10 SDK (10.0.20348.0)

~~~
test_ld cc -ID:/installed/x64-windows-static/include -LD:/installed/x64-windows-static/lib -lopenmpt -lzlib -lvorbisfile -lvorbis -logg -lmpg123 -lstdc++
test_cc -ID:/installed/x64-windows-static/include -libpath:D:/installed/x64-windows-static/lib
BEGIN ./ffconf.SiE1gC11/test.c
    1	#include <libopenmpt/libopenmpt.h>
    2	#include <stdint.h>
    3	long check_openmpt_module_create(void) { return (long) openmpt_module_create; }
    4	int main(void) { int ret = 0;
    5	 ret |= ((intptr_t)check_openmpt_module_create) & 0xFFFF;
    6	return ret; }
END ./ffconf.SiE1gC11/test.c
cl.exe -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_USE_MATH_DEFINES -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS -D_WIN32_WINNT=0x0600 -DPIC @D:/buildtrees/ffmpeg/x64-windows-static-rel/cflags.rsp -nologo -DHAVE_UNISTD_H=0 -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include/harfbuzz -ID:/installed/x64-windows-static/include/libpng16 -ID:/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include/libpng16 -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include/libpng16 -ID:/installed/x64-windows-static/include/fribidi -DFRIBIDI_LIB_STATIC -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -ID:/installed/x64-windows-static/include -libpath:D:/installed/x64-windows-static/lib -c -Fo./ffconf.SiE1gC11/test.o ./ffconf.SiE1gC11/test.c
cl : Command line warning D9002 : ignoring unknown option '-libpath:D:/installed/x64-windows-static/lib'
test.c
./ffconf.SiE1gC11/test.c(3): warning C4311: 'type cast': pointer truncation from 'openmpt_module *(__cdecl *)(openmpt_stream_callbacks,void *,openmpt_log_func,void *,const openmpt_module_initial_ctl *)' to 'long'
./ffconf.SiE1gC11/test.c(3): warning C4996: 'openmpt_module_create': was declared deprecated
link.exe @D:/buildtrees/ffmpeg/x64-windows-static-rel/ldflags.rsp -nologo -ID:/installed/x64-windows-static/include -libpath:D:/installed/x64-windows-static/lib -out:./ffconf.SiE1gC11/test.exe ./ffconf.SiE1gC11/test.o openmpt.lib zlib.lib vorbisfile.lib vorbis.lib ogg.lib mpg123.lib
LINK : warning LNK4044: unrecognized option '/ID:/installed/x64-windows-static/include'; ignored
mpg123.lib(compat.c.obj) : error LNK2019: unresolved external symbol __imp_PathCombineW referenced in function INT123_compat_catpath
mpg123.lib(compat.c.obj) : error LNK2019: unresolved external symbol __imp_PathIsRelativeW referenced in function u2wlongpath
mpg123.lib(compat.c.obj) : error LNK2019: unresolved external symbol __imp_PathIsUNCW referenced in function u2wlongpath
.\ffconf.SiE1gC11\test.exe : fatal error LNK1120: 3 unresolved externals
ERROR: libopenmpt >= 0.2.6557 not found using pkg-config
~~~
It seems that the problem starts at port `mpg123`, not `libopenmpt`.

--- a/ports/mpg123/portfile.cmake
+++ b/ports/mpg123/portfile.cmake
@@ -15,11 +15,17 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     yasm_tool_helper(APPEND_TO_PATH)
 endif()
 
+vcpkg_list(SET options)
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_list(APPEND options "-DLIBMPG123_LIBS=-lshlwapi")
+endif()
+
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}/ports/cmake"
     OPTIONS
         -DUSE_MODULES=OFF
         -DBUILD_PROGRAMS=OFF
+        ${options}
     MAYBE_UNUSED_VARIABLES
         BUILD_PROGRAMS
 )

--- a/ports/mpg123/portfile.cmake
+++ b/ports/mpg123/portfile.cmake
@@ -11,7 +11,6 @@ vcpkg_from_sourceforge(
 )
 
 if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
-    include("${CURRENT_INSTALLED_DIR}/share/yasm-tool-helper/yasm-tool-helper.cmake")
     yasm_tool_helper(APPEND_TO_PATH)
 endif()
 

--- a/ports/mpg123/vcpkg.json
+++ b/ports/mpg123/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "mpg123",
   "version": "1.31.3",
-  "port-version": 2,
+  "port-version": 3,
   "description": "mpg123 is a real time MPEG 1.0/2.0/2.5 audio player/decoder for layers 1, 2 and 3 (MPEG 1.0 layer 3 also known as MP3).",
   "homepage": "https://sourceforge.net/projects/mpg123/",
   "license": "LGPL-2.1-or-later",

--- a/ports/mpg123/vcpkg.json
+++ b/ports/mpg123/vcpkg.json
@@ -16,8 +16,12 @@
       "host": true
     },
     {
-      "name": "yasm-tool-helper",
-      "platform": "windows"
+      "name": "yasm",
+      "host": true,
+      "features": [
+        "tools"
+      ],
+      "platform": "windows & !mingw"
     }
   ]
 }

--- a/scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json
+++ b/scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json
@@ -97,14 +97,6 @@
       "name": "ffmpeg",
       "default-features": false,
       "features": [
-        "x265"
-      ],
-      "platform": "!uwp & !(arm & windows)"
-    },
-    {
-      "name": "ffmpeg",
-      "default-features": false,
-      "features": [
         "aom"
       ],
       "platform": "!(windows & arm & !uwp)"
@@ -116,6 +108,14 @@
         "dav1d"
       ],
       "platform": "!(uwp | arm | x86 | osx)"
+    },
+    {
+      "name": "ffmpeg",
+      "default-features": false,
+      "features": [
+        "x265"
+      ],
+      "platform": "!uwp & !(arm & windows)"
     },
     {
       "name": "ffmpeg",

--- a/scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json
+++ b/scripts/test_ports/vcpkg-ci-ffmpeg/vcpkg.json
@@ -23,6 +23,7 @@
         "mp3lame",
         "nonfree",
         "openjpeg",
+        "openmpt",
         "openssl",
         "opus",
         "postproc",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5586,7 +5586,7 @@
     },
     "mpg123": {
       "baseline": "1.31.3",
-      "port-version": 2
+      "port-version": 3
     },
     "mpi": {
       "baseline": "1",

--- a/versions/m-/mpg123.json
+++ b/versions/m-/mpg123.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6e990602e7caa6965db94c4ef01e3f1b44fb4f67",
+      "version": "1.31.3",
+      "port-version": 3
+    },
+    {
       "git-tree": "58fbb0bf0ec6307689125ef1ba71112b670a42a9",
       "version": "1.31.3",
       "port-version": 2

